neural_compressor/compression/pruner/patterns/ninm.py

```diff
@@ -165,7 +168,10 @@ def _reshape_2dims_to_orig(self, data, orig_shape):
         """
         if len(orig_shape) == 4:
             data = data.reshape(orig_shape[0], orig_shape[2], orig_shape[3], orig_shape[1])
-            data = data.permute(0, 3, 1, 2)
```

Comment:
Is this a mistake to overwrite the previous permute?

Reply:
Thanks! Changed